drivers/oled/oled_driver.c

```diff
@@ -431,6 +431,31 @@ void oled_write_ln(const char *data, bool invert) {
     oled_advance_page(true);
 }
 
+void oled_pan(bool left) {
+    uint16_t i = 0;
+    for (uint16_t y = 0; y < OLED_DISPLAY_HEIGHT/8; y++) {
+        if(left) {
+            for (uint16_t x = 0; x < OLED_DISPLAY_WIDTH - 1; x++) {
+                i = y * OLED_DISPLAY_WIDTH + x;
+                oled_buffer[i] = oled_buffer[i+1];
+            }
+        } else {
+            for (uint16_t x = OLED_DISPLAY_WIDTH -1; x > 0; x--) {
+                i = y * OLED_DISPLAY_WIDTH + x;
+                oled_buffer[i] = oled_buffer[i-1];
+            }
+        }
+    }
+    oled_dirty = 65535;
```

Comment:
lets make this `~((OLED_BLOCK_TYPE)0)`, just in case someone needs to bump it to 32-bit.

Reply:
Super good note, thank you @tzarc !  I've made a commit to implement this change.